tableauserverclient/server/endpoint/resource_tagger.py

```diff
@@ -0,0 +1,33 @@
+from .endpoint import Endpoint
+from .. import RequestFactory
+from ...models.tag_item import TagItem
+import logging
+import copy
+
+logger = logging.getLogger('tableau.endpoint')
+
+
+class ResourceTagger(Endpoint):
+    # Add new tags to resource
+    def _add_tags(self, baseurl, resource_id, tag_set):
```

Comment:
I am not sure I agree with this. The users of this (the Workbook, Datasource, Views endpoints all construct the baseurl for each call to allow for users to change the version number between any calls. It is not a common case but it is one we allow. We should not break that model.